linkerd/app/src/trace_collector/oc_collector.rs

```diff
@@ -0,0 +1,63 @@
+use crate::trace_collector::EnabledCollector;
+use linkerd_app_core::control::ControlAddr;
+use linkerd_app_core::http_tracing::CollectorProtocol;
+use linkerd_app_core::proxy::http::HttpBody;
+use linkerd_app_core::Error;
+use linkerd_opencensus::{self as opencensus, metrics, proto};
+use std::{collections::HashMap, time::SystemTime};
+use tokio::sync::mpsc;
+use tokio_stream::wrappers::ReceiverStream;
+use tonic::body::BoxBody;
+use tonic::client::GrpcService;
+use tracing::Instrument;
+
+const SPAN_BUFFER_CAPACITY: usize = 100;
+const SERVICE_NAME: &str = "linkerd-proxy";
```

Comment:
If these configs re common across both impls, should they pulled up into the parent trace_collector module directly andreferenced from each?

Reply:
Done.